lib/sqlalchemy/sql/selectable.py

```diff
@@ -6474,22 +6513,22 @@ class ScalarSelect(
 
     element: SelectBase
 
-    def __init__(self, element):
+    def __init__(self, element: Select[Any]) -> None:
         self.element = element
         self.type = element._scalar_type()
 
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325